en/docs/insights/choreo-insights-api.md

```diff
@@ -1162,6 +1162,207 @@ Time filter
 
 Successful API usage by application filter.
 
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>getSuccessAPIsUsageByApplicationsWithOnPremKey</strong></td>
+<td valign="top">[<a href="#successapiusagebyapplication">SuccessAPIUsageByApplication</a>]</td>
+<td>
+
+Returns a list of successful hit counts (i.e., 2xx response codes) within the specified time range for each API,
+grouped by each application. Optionally, you can configure the <code>successAPIUsageByAppFilter</code> parameter to further
+filter the results.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">onPremKey</td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Value of the On-Prem key
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">timeFilter</td>
+<td valign="top"><a href="#timefilter">TimeFilter</a>!</td>
+<td>
+
+Time filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">successAPIUsageByAppFilter</td>
+<td valign="top"><a href="#successapiusagebyappfilter">SuccessAPIUsageByAppFilter</a></td>
+<td>
+
+Successful API usage by application filter.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>getCustomReportOvertime</strong></td>
+<td valign="top"><a href="#customreportovertime">CustomReportOvertime</a></td>
+<td>
+
+Returns summary of hits/ latency data for the inquired time frame grouped into timeSpans
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">dataFilter</td>
+<td valign="top"><a href="#datafilter">DataFilter</a>!</td>
+<td>
+
+Data filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">timeFilter</td>
+<td valign="top"><a href="#timefilter">TimeFilter</a>!</td>
+<td>
+
+Time filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">metric</td>
+<td valign="top"><a href="#metric">Metric</a>!</td>
+<td>
+
+Metric to retrieve data for
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">groupByFields</td>
+<td valign="top">[<a href="#groupbyfield">GroupByField</a>]!</td>
+<td>
+
+Column/ Field to group the data by
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">groupByValues</td>
+<td valign="top">[<a href="#string">String</a>]!</td>
+<td>
+
+Value to filter by on the groupByField
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">granularity</td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+The granularity value for which data is retrieved.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>getCustomReportTopOvertime</strong></td>
+<td valign="top"><a href="#customreporttopovertime">CustomReportTopOvertime</a></td>
+<td>
+
+Returns summary of hits/ latency data for the inquired time frame grouped into timeSpans for the top N values
```

Comment:
See comment on line 1210.

Reply:
Responded in the relevant comment thread